doc/cprover-manual/cbmc-tutorial.md

```diff
@@ -138,11 +138,11 @@ possible output format.
 ```
 
 The specification of the XML trace output can be found here: [XML
```

Comment:
A tutorial should absolutely not have a pointer to the XML spec.

Reply:
Done (as a second commit).